apps/hash-ai-worker-ts/src/activities/shared/find-existing-entity.ts

```diff
@@ -124,7 +124,7 @@ export const findExistingEntity = async ({
   /**
    * First find suitable specific properties to match on
    */
-  const labelProperty = entityType.labelProperty;
+  const labelProperty = entityType.labelProperty as BaseUrl | undefined;
```

Comment:
An explicit cast is required because `BaseUrl` in the type system is an alias to `string`.